### DISCUSSION TITLE
using pre commit hook

### DISCUSSION THREAD

**Discussion by fakeid30** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#149] [Ideas]
Some hooks in this repo : https://github.com/pre-commit/pre-commit-hooks looks interesting. Will be nice to have a couple of hooks. The hooks I like are: 

- [check-symlink](https://github.com/pre-commit/pre-commit-hooks#check-symlinks)
- [check-docstring-first](https://github.com/pre-commit/pre-commit-hooks#check-docstring-first)
- [check-toml](https://github.com/pre-commit/pre-commit-hooks#check-toml)
- [destroyed-symlinks](https://github.com/pre-commit/pre-commit-hooks#destroyed-symlinks)
- [detect-private-key](https://github.com/pre-commit/pre-commit-hooks#detect-private-key)
- [trailing-whitespace](https://github.com/pre-commit/pre-commit-hooks#trailing-whitespace)

@cullzie what do you think ?

**Comment by cullzie** [comment]
I don't have any preferences on this to be honest. Black and isort take care of most of the possible issues we might have but we can experiment with these and see if they offer any benefits

**Reply by fakeid30** [reply]
It's not a must have by any mean. Just nice to have . But yes I agree. Black & isort are doing most of the things here.
